dts/bindings/pwm/st,stm32-pwm.yaml

```diff
@@ -10,8 +10,12 @@ properties:
 
     st,prescaler:
       type: int
-      required: true
-      description: Clock prescaler at the input of the timer
+      required: false
+      deprecated: true
+      description: |
+        Clock prescaler at the input of the timer
+        Note: This property is obsolete and was replaced by "st,prescaler" at
+        timers level. It is kept temporarily for compatibility reasons.
```

Comment:
Lest' be more specific about "timer level".
It can be replaced by "binding of parent timer node" or something similar